### PR TITLE
Adds JSCS - and formats settings js

### PR DESCRIPTION
**Pull request by Gomez**
This adds the jscs config file with sane defaults. I formatted the settings.js. 

See #631

### CODE REVIEW

**Comment by Gomez** [comment]
@DeepDiver1975 Gone through all PRs. settings.js is not touched. Should be safe to merge this PR. travis will be enabled as we are done with errors...

File: js/settings.js [hunk 1]
```diff
@@ -1,32 +1,29 @@
...
 			data: {accountId:accountId},
 			type: 'DELETE',
-			success: function () {
-                // reload the complete page
-                // TODO should only reload the app nav/content
-                window.location.reload();
+			success: function() {
+				// reload the complete page
+				// TODO should only reload the app nav/content
+				// window.location.reload();
```
Comment (jancborchardt) [inline js/settings.js]: This is the only actual code change - commenting out the line. Was this really intended?

Comment (Gomez) [inline js/settings.js]: Good catch. Reload must happen here. I fixed it and squash to one commit.

**Comment by Gomez** [comment]
@DeepDiver1975 @wurstchristoph @clementhk @Xenopathic Take a look at the "Friendly packages" here: http://jscs.info/overview.html
Very helpful, would be cool to get the syntax style everywhere applied.

**Comment by DeepDiver1975** [comment]
let me have a look at this these days ... do the jscs rules conflict with jshint?

**Comment by Gomez** [comment]
No conflicts.
JSHint recommends using JSCS for style related tooling. Indentation checking is/will be removed from jshint...
(Check out the phpstrom plugin :))

**Comment by clementhk** [comment]
I've heard of it and all for it :)

File: js/settings.js [hunk 2]
```diff
@@ -1,32 +1,29 @@
...
-                // reload the complete page
-                // TODO should only reload the app nav/content
-                window.location.reload();
+			success: function() {
+				// reload the complete page
+				// TODO should only reload the app nav/content
+				window.location.reload();
 			},
-            error: function() {
+			error:function() {
```
Comment (clementhk) [inline js/settings.js]: Can you keep the space after : so it is same as above?

Comment (Gomez) [inline js/settings.js]: Done.

**Comment by Gomez** [comment]
One more to merge? @DeepDiver1975

**Comment by clementhk** [comment]
reviewed by me.

**Comment by DeepDiver1975** [comment]
:+1:

**Pull request merged**